### PR TITLE
img py36

### PR DESCRIPTION
**Pull request by jpizarrom**

### CODE REVIEW

File: Dockerfile.amazonlinux3.6 [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    blas-devel \
+    gcc \
+    gcc-c++ \
+    lapack-devel \
+    findutils \
+    zip
+
+RUN yum -y install https://rhel6.iuscommunity.org/ius-release.rpm
+RUN yum -y install \
+    python36u \
```
Comment (jpizarrom) [inline Dockerfile.amazonlinux3.6:16]: py36

File: Dockerfile.amazonlinux3.6 [hunk 2]
```diff
@@ -0,0 +1,24 @@
+FROM amazonlinux:2016.09
```
Comment (jpizarrom) [inline Dockerfile.amazonlinux3.6:1]: py36 dockerfile

File: build3.6.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```
Comment (jpizarrom) [inline build3.6.sh:1]: build py36

File: build3.6.sh [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    /usr/bin/python3.6 -m venv --copies /sklearn_build
+    source /sklearn_build/bin/activate
+
+    do_pip
+
+    shared_libs
+
+    strip_virtualenv
+
+    uploadtos3
```
Comment (jpizarrom) [inline build3.6.sh:45]: uploadtos3

File: docker-compose.yml [hunk 5]
```diff
@@ -9,3 +9,13 @@
     volumes:
        - .:/outputs
     command: /bin/bash /outputs/build.sh
+
+
+  amazonlinux3.6:
```
Comment (jpizarrom) [inline docker-compose.yml:14]: amazonlinux3.6:

File: requirements.sh [hunk 6]
```diff
@@ -4,5 +4,10 @@ set -ex
 pip install --upgrade pip wheel
 pip install --use-wheel --no-binary numpy numpy==1.11.1
 pip install --use-wheel --no-binary scipy scipy==0.18.1
-pip install --use-wheel scikit-learn==0.18
 pip install --use-wheel joblib==0.11
+
+pip install --use-wheel pillow==4.2.1
```
Comment (jpizarrom) [inline requirements.sh:9]: img deps

File: build3.6.sh (hunk 4 above)
Comment (jpizarrom) [reply build3.6.sh]: https://console.aws.amazon.com/s3/buckets/admetricks-lambda-functions/sklearn-build-lambda/?region=us-east-1&tab=overview

**Review by pdelsol — APPROVED** [review]

**Pull request closed without merging**